stats/report.py

```diff
@@ -22,10 +22,11 @@ def show_report(repo):
     print(sep)
 
     print("\n* Repo changes (inserts and deletes) per week:\n")
-    for week, changes in sorted(
-        gstats.get_number_of_changes_per_week()
-    ):
-        print(f"{week:<38} | {changes:>3}")
+
+    changes_dict = sorted(gstats.get_number_of_changes_per_week())
```

Comment:
I usually don't put the type in variable names, you can use type hints for that.

Reply:
Makes sense! I switched `changes_dict` to `sorted_changes`